examples/slash_commands/injections.py

```diff
@@ -41,7 +41,7 @@ async def get_config(
     """
     # if a locale is not provided use the guild's locale
     if locale is None:
-        locale = inter.guild and inter.guild.preferred_locale or "en-US"
+        locale = inter.guild and str(inter.guild.preferred_locale) or "en-US"
```

Comment:
instead of `"en-US"` this should use `Locale.en_US`

Reply:
Actually, looking more into this example, we should remove it. It gets a locale from a user provided locale, which now automatically provided by discord. I think this is fine for now.